source/content/guides/decoupled/wp-gatsby-frontend-starters/02-create.md

```diff
@@ -72,29 +35,30 @@ Make sure you meet the prerequisites below before you continue.
 
 * You have installed your WordPress site using the [Decoupled WordPress Recommended Project](https://github.com/pantheon-upstreams/decoupled-wordpress-composer-managed) configuration, which has the `wp-graphql` and `wp-gatsby` plugins enabled.
 
+1. Log in to your **Pantheon Dashboard** and select the **Sites** page.
+
+1. Click the **Decoupled** tab and then click **+Create New Site**.
+
+    <Alert title="Note"  type="info" >
+
+    You cannot use a . (period) or _ (underscore) for site and Multidev names.
+
+    </Alert>
+
 1. Click the **Gatsby + WordPress** template, select your Git provider under **Choose your Git provider**, and then click **Continue**.
 
+  ![select a starter](../../../../images/decoupled-select-starter.png)
+
 1. Select your desired GitHub account and enter the repository name. You can select **Make this a private repository** to provide greater security for your Git repository.
 
-1. Select the WordPress CMS backend from the drop-down menu.
+1. Select the desired WordPress CMS backend from the **Link Your CMS** drop-down menu.
```

Comment:
Should there be another optional step here to point out the Environment Variables section in the "Advanced Settings" Tab during site creation?

Reply:
Added! Thank you!